tests/data-type.js

```diff
@@ -141,6 +141,27 @@ test('test validity of `createdBy` field from another peer', async (t) => {
   await destroy()
 })
 
+test('delete()', async (t) => {
+  t.plan(4)
```

Comment:
not really needed for a test like this. plan is mostly useful when dealing with things like event emitters or where the execution of code is not sequential

```suggestion
```